persistence/pg.go

```diff
@@ -37,6 +38,9 @@ type InvoiceCreationData struct {
 type dbInvoice struct {
 	tableName struct{} `pg:"lnmux.invoices,discard_unknown_columns"` // nolint
 
+	// SequenceNum is a unique identifier used for pagination
+	SequenceNum uint64 `pg:"sequence_num,pk"`
```

Comment:
Storing the sequence number and adding the query rpc can be separate commits.

Reply:
Hmm, the primary key is defined differently here (seq+hash) than it is in the sql script.

Reply:
Right, I will remove the pk from Sequence number